Source/WebCore/platform/graphics/ImageUtilities.h

```diff
@@ -30,6 +30,9 @@
 
 namespace WebCore {
 
+class IntSize;
+class Settings;
```

Comment:
I think this is layering variation. Settings should not be referenced in the platform directory.

Reply:
Moved to `SVGImage`.